pkgs/desktops/gnome-3/extensions/text-translator/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, gnome3, pkgs, substituteAll, translate-shell}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-text-translator";
+  version = "unstable-2020-02-25";
+
+  src = fetchFromGitHub {
+    owner = "gufoe";
+    repo = "text-translator";
+    rev = "991daadff538bd7acd15cf60415704e0215f2bf6";
+    sha256 = "098a967p1gr8g20nz0fdnlmpc6k40fh0ijqcdhp7ckfqr39imkhr";
+  };
+
+  uuid = "text_translator@awamper.gmail.com";
+
+  patches = [
+    (substituteAll {
+      src = ./fix-trans.patch;
+      trans = "${translate-shell}/bin/trans";
+    })
+  ];
+
+  nativeBuildInputs = with pkgs; [
```

Comment:
Why are these in `nativeBuildInputs`?

Reply:
Should I change to `buildInputs`?

Reply:
Generally, run-time dependencies should go to `buildInputs`, though for extensions that does not work. See https://nixos.org/nixpkgs/manual/#ssec-gnome-common-issues-unwrappable-package for a solution.

Reply:
Thanks @jtojnar, I'll fix it!

Reply:
@jtojnar, I think this package doesn't need this buildInputs, I removed them and built the derivation again in qemu and it seems to work correctly, should I just remove these inputs? The extension doesn't seems to need it, right?
I just added because I thought it was really necessary checking [here](https://github.com/gufoe/text-translator/blob/master/extension.js#L14), am I missing something?

Reply:
Or maybe I should change [here](https://github.com/gufoe/text-translator/blob/master/extension.js#L6), but I don't know what path set and why without changing this, does it works on emulator.

Reply:
Searching the source code, it only seems to use `Clutter`, not `ClutterGst` or `GtkClutter`, so the README is probably incorrect. And `Clutter` is provided by Mutter, so it is already available to GNOME Shell. That might explain why you do not need to add it explicitly.

Reply:
But you might still need to add `${mutter}/lib/mutter-5`  `${gnome3.mutter}/lib/mutter-5` or `${clutter}/lib/girepository-1.0` when opening the preferences from GNOME Tweaks, which does not run in GNOME Shell context.

Reply:
Nice, got it!
Removed these dependencies, thanks @jtojnar!

Reply:
Does opening preferences from GNOME Tweaks work then? Might want to run them from Terminal and check the console output.

Reply:
It seems to work: 
![image](https://user-images.githubusercontent.com/7820865/76709958-f908ce00-66e1-11ea-9c95-844142ebdb28.png)

How can I test it from terminal?